Sources/SourceKitLSP/DocumentManager.swift

```diff
@@ -308,11 +270,21 @@ extension DocumentManager {
     }
   }
 
-  /// Convenience wrapper for `edit(_:newVersion:edits:editCallback:)` that logs on failure.
+  /// Convenience wrapper for `edit(_:newVersion:edits:beforeCallback:)` that logs on failure.
```

Comment:
Nitpick: The other function now also has an `afterCallback:` so this should be 
`edit(_:newVersion:edits:beforeCallback:afterCallback:)`